Sources/eppo/EppoClient.swift

```diff
@@ -103,9 +99,9 @@ public class EppoClient {
 
         if subjectKey.count == 0 { throw Errors.subjectKeyRequired }
         if flagKey.count == 0 { throw Errors.flagKeyRequired }
-        if self.flagConfigs.flags.count == 0 { throw Errors.configurationNotLoaded }
+        if !self.configurationStore.isInitialized() { throw Errors.configurationNotLoaded }
```

Comment:
Are we expecting that SDK usage will always involve additional exception handling to prevent these thrown errors from breaking client applications?

Reply:
graceful mode is needed on swift https://linear.app/eppo/issue/FF-1953/add-graceful-mode-to-ios-sdk